packages/lit-dev-content/samples/checkable-tutorial-base.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "/samples/base.json",
+  "extends": "/samples/v2-base.json",
```

Comment:
Would it be easier for tutorials and non-doc samples to keep a `base.json` that we can update to latest version all together? We'd likely want to make those evergreen samples work with the latest version of Lit.

Reply:
That is a great suggestion, and that `base.json` can extend whichever version we want to be the latest. I like it!

Reply:
Great suggestion! Applied!